content/aips/RETHETHV3.md

```diff
@@ -86,4 +86,8 @@ The proposal Payload was reviewed by[Bored Ghost Developing](https://bgdlabs.com
 
 ## Copyright
 
+<<<<<<< HEAD
```

Comment:
there should be no changes here (`git fetch origin && git checkout main content/aips/RETHETHV3.md`)